lib/modules/manager/gomod/update.ts

```diff
@@ -137,7 +137,10 @@ export function updateDependency({
     }
 
     if (depType === 'indirect') {
-      newLine += ' // indirect';
+      newLine = newLine.replace(
+        regEx(/(?:\s*\/\/\s*indirect\s*)$/),
+        ' // indirect'
```

Comment:
what if there's a `// indirect;` ? then we would append again?

Reply:
Added to the regexp